keras_hub/src/models/resnet/resnet_presets.py

```diff
@@ -79,4 +79,145 @@
         },
         "kaggle_handle": "kaggle://kerashub/resnetv2/keras/resnet_v2_101_imagenet/3",
     },
+    "resnet_vd_18_imagenet": {
+        "metadata": {
+            "description": (
+                "18-layer ResNetVD (ResNet with bag of tricks) model "
+                "pre-trained on the ImageNet 1k dataset at a 224x224 "
+                "resolution."
+            ),
+            "params": 11722824,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet18_vd",
+    },
+    "resnet_vd_34_imagenet": {
+        "metadata": {
+            "description": (
+                "34-layer ResNetVD (ResNet with bag of tricks) model "
+                "pre-trained on the ImageNet 1k dataset at a 224x224 "
+                "resolution."
+            ),
+            "params": 21838408,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet34_vd",
+    },
+    "resnet_vd_50_imagenet": {
+        "metadata": {
+            "description": (
+                "50-layer ResNetVD (ResNet with bag of tricks) model "
+                "pre-trained on the ImageNet 1k dataset at a 224x224 "
+                "resolution."
+            ),
+            "params": 25629512,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet50_vd",
+    },
+    "resnet_vd_50_ssld_imagenet": {
+        "metadata": {
+            "description": (
+                "50-layer ResNetVD (ResNet with bag of tricks) model "
+                "pre-trained on the ImageNet 1k dataset at a 224x224 "
+                "resolution with knowledge distillation."
+            ),
+            "params": 25629512,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet50_vd_ssld",
+    },
+    "resnet_vd_50_ssld_v2_imagenet": {
+        "metadata": {
+            "description": (
+                "50-layer ResNetVD (ResNet with bag of tricks) model "
+                "pre-trained on the ImageNet 1k dataset at a 224x224 "
+                "resolution with knowledge distillation and AutoAugment."
+            ),
+            "params": 25629512,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet50_vd_ssld_v2",
+    },
+    "resnet_vd_50_ssld_v2_fix_imagenet": {
```

Comment:
what is `fix`?

Reply:
"`Fix_ResNet50_vd_ssld_v2` stopped all parameter updates of ResNet50_vd_ssld_v2 except the FC layer,and fine-tuned on ImageNet1k dataset" (from https://paddleclas.readthedocs.io/en/latest/models/ResNet_and_vd_en.html)

Reply:
thanks!